pkg/gui/gui.go

```diff
@@ -192,14 +209,28 @@ func (gui *Gui) layout(g *gocui.Gui) error {
 		optionsTop = height - 1
 	}
 
-	v, err := g.SetView("main", leftSideWidth+panelSpacing, 0, width-1, optionsTop, gocui.LEFT)
-	if err != nil {
-		if err != gocui.ErrUnknownView {
-			return err
+	isStreamingStatusEnable := gui.Config.GetUserConfig().GetBool("commandStatus")
+
+	if isStreamingStatusEnable {
```

Comment:
this duplicates some code. The only difference between the two blocks in the if statement is `commitsBranchesBoundary` vs `optionsTop`, so we can just assign that value to a variable based on the value of `isStreamingStatusEnable` and then we don't need to have the SetView method repeated